### PR TITLE
initial update to actions of the steering board

### PR DESCRIPTION
**Pull request by niallrobinson**
`nebulous` didn't go down well upon user testing

### CODE REVIEW

**Comment by rabernat** [comment]
This section is considerably shorted and less detailed than the jupyter counterpart: https://github.com/jupyter/governance/blob/master/governance.md#steering-council

Was that intentional?

Maybe we should define the roles and responsibilities more clearly.

**Comment by niallrobinson** [comment]
no, it wasn't :D - thanks for the link, another commit coming up

File: governance.md [hunk 1]
```diff
@@ -11,6 +11,25 @@ https://github.com/pangeo-data/governance
 
 ### The Steering Council
 
+The Project will have a Steering Council that consists of Project Contributors who have produced contributions that are substantial in quality and quantity, and sustained over at least one year. The overall role of the Council is to ensure, through taking input from the Community, the long-term well-being of the project, both technically and as a community. As Pangeo draws its strength from representing the Community, it strives to avoid fragmentation of the comminity into separate projects, if possible.
```
Comment (jhamman) [inline governance.md]: As Pangeo draws --> As The Project draws

File: governance.md [hunk 2]
```diff
@@ -11,6 +11,25 @@ https://github.com/pangeo-data/governance
...
+
+During the everyday project activities, council members participate in all discussions, code review and other project activities as peers with all other Contributors and the Community. In these everyday activities, Council Members do not have any special power or privilege through their membership on the Council. However, it is expected that because of the quality and quantity of their contributions and their expert knowledge of the Project Software and Services that Council Members will provide useful guidance, both technical and in terms of project direction, to potentially less experienced contributors.
```
Comment (jhamman) [inline governance.md]: Title case "council members" in sentence 1. Also, I think members "may" participate in specific project activities.

File: governance.md [hunk 3]
```diff
@@ -11,6 +11,25 @@ https://github.com/pangeo-data/governance
...
+
+The Steering Council and its Members play a special role in certain situations. In particular, the Council may:
+
+Make decisions about the overall scope, vision and direction of the project.
+Make decisions about strategic collaborations with other organizations or individuals.
+Make decisions about specific technical issues, features, bugs and pull requests. They are the primary mechanism of guiding the code review process and merging pull requests.
+Make decisions about the Services that are run by The Project and manage those Services for the benefit of the Project and Community.
+
+Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
```
Comment (jhamman) [inline governance.md]: suggest we make this section a bulleted list

File: governance.md [hunk 4]
```diff
@@ -11,6 +11,25 @@ https://github.com/pangeo-data/governance
...
+
+Outcomes and decisions should be clearly communicated to the community. This can take three different forms: 
+
+* New challenges for Pangeo will be detailed as a github issue for the community to discuss.
```
Comment (jhamman) [inline governance.md]: github --> GitHub

File: governance.md [hunk 5]
```diff
@@ -11,6 +11,25 @@ https://github.com/pangeo-data/governance
...
+* Resolved technical deadlocks will result in an explanation on the relevant issue and/or closure of the relevant pull request.
+* Other decisions will be detailed as a narrative blog post.
```
Comment (rabernat) [inline governance.md]: What about confidential / private matters? We should make this more flexible. The openness described above is a goal, but it should not be strictly required.

File: governance.md (hunk 1 above)
Comment (rabernat) [reply governance.md]: Some of this is duplicative of (and in conflict with) #1, which specifies criteria for membership on the council.

File: governance.md (hunk 3 above)
Comment (rabernat) [reply governance.md]: - coordinate fundraising activities amongst institutions?

**Comment by mcgibbon** [comment]
As it is currently written, this document is [strongly masculine-coded](http://gender-decoder.katmatfield.com/results/e1531905-1a00-4e2d-a116-9282897691f4). It risks putting women off applying, but will probably encourage men to apply ([paper](http://gender-decoder.katmatfield.com/static/documents/Gaucher-Friesen-Kay-JPSP-Gendered-Wording-in-Job-ads.pdf)). We should avoid coding the description of the steering committee to be masculine-biased in pursuit of diversity as is being discussed in #8. If you like I can make particular suggestions on changes to correct this, or @niallrobinson could make some changes focusing on masculine-coded words, phrases, and concepts and increasing the emphasis of the collaborative aspect of Pangeo.

**Comment by niallrobinson** [comment]
thanks for the feedback all - I propose that we address the remaining comments when this is merged into a whole document

**Pull request merged**